src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -282,4 +282,15 @@ describe('Facility search', () => {
 
     cy.axeCheck();
   });
+
+  it('renders static map images on detail page', () => {
+    // from https://stackoverflow.com/questions/51246606/test-loading-of-image-in-cypress
+    cy.visit('/find-locations/facility/vha_688GA');
+    cy.get('[alt="Static map"]')
```

Comment:
There's `cy.findByAltText` and `cy.findAllByAltText` from Cypress Testing Library, by the way.